demos/notebooks/demo_online_cnmfE.ipynb

```diff
@@ -78,7 +107,7 @@
    "source": [
     "# motion correction parameters\n",
     "motion_correct = True            # flag for performing motion correction\n",
-    "pw_rigid = False                 # flag for performing piecewise-rigid motion correction (otherwise just rigid)\n",
+    "pw_rigid = True                  # flag for performing piecewise-rigid motion correction (otherwise just rigid)\n",
```

Comment:
Did you mean to change this (and in this diff) or was this just for debugging??

Reply:
Good eye that was for debugging. I'll switch it back